packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -393,46 +401,32 @@ const AccountTransferForm = observer(
             is_dxtrade_allowed,
             setSideNotes,
             is_crypto,
-            internal_remaining_transfers?.allowed,
-            mt5_remaining_transfers?.allowed,
-            dxtrade_remaining_transfers?.allowed,
             is_dxtrade_transfer,
             is_mt_transfer,
             is_from_derivgo,
-            ctrader_remaining_transfers?.allowed,
             is_ctrader_transfer,
+            internal_remaining_cumulative_transfers?.allowed,
+            mt5_remaining_cumulative_transfers?.allowed,
+            ctrader_remaining_cumulative_transfers?.allowed,
+            dxtrade_remaining_cumulative_transfers?.allowed,
+            derivez_remaining_cumulative_transfers?.allowed,
         ]);
 
         React.useEffect(() => {
             const getRemainingTransfers = () => {
                 if (is_mt_transfer) {
-                    return mt5_remaining_transfers?.available;
+                    return mt5_remaining_cumulative_transfers?.available;
```

Comment:
Shouldn't we be using `mt5_remaining_cumulative_transfers?.available` only if `is_cumulative_transfer_enabled` is true? Otherwise the fallback should be mt5_remaining_transfers?.available, right?

Reply:
@nijil-deriv  yes true, I've updated the useEffect to show the message when transfer count = 0 or when cumulative limit is enabled and transfer amount = 0